File: C++/Arrey/linearSearch.c
#include <stdio.h>
int linearSearch(int arr[], int size, int key);
int main(int argc, char const *argv[])
{
    int arr[10];
    printf("enter the values of this arrey indxes: ");
    for (int i = 0; i < 10; i++)
    {
        scanf("%d", &arr[i]);
    }
    
    int func = linearSearch(arr, 10, 5);
    printf("the index of key value is: %d", func);
    
    return 0;
}

int linearSearch(int arr[], int size, int key){
    for (int i = 0; i < size; i++)
    {
        if (key == arr[i])
        {
            return i;
        }
        
    }
    
}

File: C++/Arrey/linearSearch.py
def linearSearch(arr, size, key):
    for i in range(0, size):
        if ( arr[i] == key):
            return i
    return -1

arr = []
for i in range(0,10):
    arr.append(int(input()))

lidexvalue= linearSearch(arr, 10, 5)

if indexvalue == -1:
    print("value not exist")
else:
    print("here is the index value of that arr: ", lidexvalue)


